cli/cmd/wallet/cmd/account/handler/handler_create.go

```diff
@@ -256,19 +254,16 @@ func SaveHighestData(acc core.ValidatorAccount, store *inmemory.InMemStore, acco
 	}
 
 	// add minimal attestation protection data
-	minimalAtt := &eth.AttestationData{
-		Source: &eth.Checkpoint{Epoch: types.Epoch(accountFlags.highestSources[highestIndex])},
-		Target: &eth.Checkpoint{Epoch: types.Epoch(accountFlags.highestTargets[highestIndex])},
+	minimalAtt := &phase0.AttestationData{
+		Source: &phase0.Checkpoint{Epoch: phase0.Epoch(accountFlags.highestSources[highestIndex])},
+		Target: &phase0.Checkpoint{Epoch: phase0.Epoch(accountFlags.highestTargets[highestIndex])},
 	}
 	if err := store.SaveHighestAttestation(acc.ValidatorPublicKey(), minimalAtt); err != nil {
 		return errors.Wrap(err, "failed to save highest attestation")
 	}
 
 	// add minimal proposal protection data
-	minimalProposal := &eth.BeaconBlock{
-		Slot: types.Slot(accountFlags.highestProposals[highestIndex]),
-	}
-	if err := store.SaveHighestProposal(acc.ValidatorPublicKey(), minimalProposal); err != nil {
+	if err := store.SaveHighestProposal(acc.ValidatorPublicKey(), phase0.Slot(accountFlags.highestProposals[highestIndex])); err != nil {
```

Comment:
@lior-blox Needs to check

Reply:
@olegshmuelov that could be nil or invalid position

Reply:
there is a validation here
https://github.com/bloxapp/eth2-key-manager/blob/7ac792f909a50befea65efe2fe2dfb0f2df233ad/cli/cmd/wallet/cmd/account/handler/handler_create.go#L273